build.gradle

```diff
@@ -4,6 +4,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.google.dagger:hilt-android-gradle-plugin:2.40.5'
+        classpath 'gradle.plugin.com.onesignal:onesignal-gradle-plugin:[0.12.10, 0.99.99]'
```

Comment:
I think we can just specify an exact version and then update if needed

Reply:
How would we do this?

Reply:
replace `[0.12.10, 0.99.99]` with just `0.12.10`

Reply:
actually it looks like the latest is `0.14.0` see https://github.com/OneSignal/OneSignal-Gradle-Plugin/releases